modules/osdk-updating-128-to-131.adoc

```diff
@@ -0,0 +1,243 @@
+endif::[]
+ifeval::["{context}" == "osdk-java-updating-projects"]
+:java:
+:type: Java
+endif::[]
+
+:_content-type: PROCEDURE
+[id="osdk-upgrading-projects_{context}"]
+= Updating {type}-based Operator projects for Operator SDK {osdk_ver}
+
+The following procedure updates an existing {type}-based Operator project for compatibility with {osdk_ver}.
+
+.Prerequisites
+
+* Operator SDK {osdk_ver} installed
+* An Operator project created or maintained with Operator SDK {osdk_ver_n1}
+
+.Procedure
+
+ifdef::golang,hybrid,java[]
+* Edit your Operator project's makefile to update the Operator SDK version to {osdk_ver} as shown in the following example:
++
+.Example makefile
+[source,make]
+----
+# Set the Operator SDK version to use. By default, what is installed on the system is used.
+# This is useful for CI or a project to utilize a specific version of the operator-sdk toolkit.
+OPERATOR_SDK_VERSION ?= v{osdk_ver} <1>
+----
+<1> Change the version from `{osdk_ver_n1}` to `{osdk_ver}`.
+endif::[]
+
+ifdef::helm[]
+. Edit your Operator's Dockerfile to update the Helm Operator version to {osdk_ver} as shown in the following example:
++
+.Example Dockerfile
+[source,docker]
+----
+FROM quay.io/operator-framework/helm-operator:v{osdk_ver} <1>
+----
+<1> Update the Helm Operator version from `{osdk_ver_n1}` to `{osdk_ver}`
+
+. Edit your Operator project's makefile to update the Operator SDK to {osdk_ver} as shown in the following example:
++
+.Example makefile
+[source,make]
+----
+# Set the Operator SDK version to use. By default, what is installed on the system is used.
+# This is useful for CI or a project to utilize a specific version of the operator-sdk toolkit.
+OPERATOR_SDK_VERSION ?= v{osdk_ver} <1>
+----
+<1> Change the version from `{osdk_ver-n1}` to `{osdk_ver}`.
+
+. If you use a custom service account for deployment, define the following role to require a watch operation on your secrets resource as shown in the following example:
++
+.Example `<?>.yaml` file
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1 //is this correct?
+kind: ClusterRoleBinding
+metadata:
+  name: <operator_name>-admin
+subjects:
+- kind: ServiceAccount
+  name: <operator_name>
+  namespace: <operator_namespace>
+roleRef:
+  kind: ClusterRole
+  name: cluster-admin
+  apiGroup: ""
+rules: <1>
+  - apiGroups:
+      - ""
+    resources:
+      - secrets
+    verbs:
+      - watch
+----
+<1> Add the `rules` stanza to create a watch operation for your secrets resource.
+endif::[]
+
+ifdef::ansible[]
+
+. Edit your Operator's Dockerfile to update the Ansible Operator version to {osdk_ver} as shown in the following example:
++
+.Example Dockerfile
+[source,docker]
+----
+FROM quay.io/operator-framework/ansible-operator:v{osdk_ver} <1>
+----
+<1> Update the Ansible Operator version from `{osdk_ver_n1}` to `{osdk_ver}`
++
+[WARNING]
+----
+The update to Ansible 2.15.0 in version 1.30.0 of the Ansible Operator removed the following pre-installed Python modules:
```

Comment:
Does we also have to manually update dependencies in the `[Pipfile](https://github.com/operator-framework/operator-sdk/pull/6483/files#diff-85c2285c39846cfb1ab26c4373213d7c91c4705e10d590b41c4d9503127fe812)`? Or are these version bumps handled by updating the SDK and/or Ansible Operator version?

Reply:
The version bumps should be handled by updating the ansible operator base image version